searchlib/src/vespa/searchlib/docstore/logdatastore.cpp

```diff
@@ -335,9 +335,11 @@ LogDataStore::getMaxBucketSpread() const
 {
     double maxSpread(1.0);
     MonitorGuard guard(_updateLock);
-    for (const auto & fc : _fileChunks) {
-        if (fc) {
-            if (_bucketizer && fc->frozen()) {
+    for (FileId i(0); i < FileId(_fileChunks.size()); i = i.next()) {
+        /// Ignore the the active file
```

Comment:
Consider adding a comment on why this is necessary.

Reply:
Fixed